cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractStructuredMetadataTest.java

```diff
@@ -147,10 +147,12 @@ public void testUpdateField() throws Exception {
         ApiResponse fieldResult = addFieldToAccount(metadataField);
         assertNotEquals("new_def", fieldResult.get("default_value"));
         metadataField.setDefaultValue("new_def");
+        metadataField.setDefaultDisabled(false);
```

Comment:
Add or check true value?

metadataField.setDefaultDisabled(false) - seems to be server default